.github/workflows/automate-changelog-PR.yml

```diff
@@ -0,0 +1,48 @@
+name: "Update changelog and open a pull request"
+
+on:
+  workflow_dispatch:
+    inputs:
+      fname_changelog:
+        description: 'Changelog filename'
+        required: true
+        default: 'CHANGES.md'
+      milestone_title:
+        description: 'Milestone title'
+        required: true
+
+jobs:
+  create-changelog-PR:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Python (to install 'changelog')
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Checkout 'changelog'
+      uses: actions/checkout@v2
+      with:
+        repository: neuropoly/changelog
+        path: changelog
+    - name: Install 'changelog'
+      run: |
+        pip install ./changelog
+    - name: Checkout '${{ github.event.repository.name }}'
+      uses: actions/checkout@v2
+      with:
+        path: ${{ github.event.repository.name }}
+    - name: Update '${{ github.event.inputs.fname_changelog }}' for '${{ github.event.repository.name }}'
+      run: |
+        cd ${{ github.event.repository.name }}
+        changelog neuropoly/spinalcordtoolbox --update --name "${{ github.event.inputs.fname_changelog }}" --milestone ${{ github.event.inputs.milestone_title }}
+        rm "${{ github.event.inputs.fname_changelog }}.bak"
```

Comment:
Huh, why? We have git to serve as a backup system. Why is changelog duplicating that work?

https://github.com/neuropoly/changelog/pull/27

Reply:
Good catch. :smile:

Reply:
I should probably change this to `rm -f` that way the workflow doesn't fail abruptly if https://github.com/neuropoly/changelog/pull/27 is addressed.